packages/database/src/services/migrations/constants.ts

```diff
@@ -18,8 +18,11 @@ export const NON_SYNCING_TABLES = [
 ];
 
 export const NON_LOGGED_TABLES = [
+  // logs
   'logs.changes',
+  'logs.accesses',
   'logs.debug_logs',
```

Comment:
Is there a reason we're not excluding all tables in the `logs` schema? There's also the `fhir_writes` table and `migrations` table that I'm a bit surprised we'd be logging